draft-davidson-pp-architecture.md

```diff
@@ -93,933 +77,234 @@ informative:
-malicious server behavior.
-
-## Key revocation
-
-Currently, key revocation is only supported by rotating the current
-configuration. In the future, we may consider adding an explicit
-mechanism for revoking a specific configuration (for example the active
-redemption configuration referred to by `server_id.previous`) without
-rotating the current issuance key.
+Clients may also choose to publicly expose registries that they do not
+trust. This could be done by posting on some public bulletin board, or
+via other means such as Client gossip protocols.
 
 # Server running modes {#running-modes}
 
 We provide an overview of some of the possible frameworks for
-configuring the way that servers run in the Privacy Pass ecosystem. In
-short, servers may be configured to provide symmetric issuance and
-redemption with clients. While some servers may be configured as proxies
-that accept Privacy Pass data and send it to another server that
-actually processes issuance and/or redemption data.
+configuring the way that Servers run in the Privacy Pass ecosystem. In
+short, Servers may be configured to provide symmetric issuance and
+redemption with clients. While some Servers may be configured as proxies
+that accept Privacy Pass data and send it to another Server that
+actually processes issuance and/or redemption data. Finally, we also
+consider instances of the protocol that may permit public verification.
 
 The intention with providing each of these running modes is to cover the
 different applications that utilize variants of the Privacy Pass
-protocol. We RECOMMEND that any Privacy Pass server implementation
+protocol. We RECOMMEND that any Privacy Pass Server implementation
 adheres to one of these frameworks.
 
 ## Single-Issuer Single-Verifier {#sisv}
 
 The simplest way of considering the Privacy Pass protocol is in a
 setting where the same server plays the role of issuer and verifier, we
-call this "Single-Issuer Single-Verifier" (SISV). In SISV, we consider a
-server S that publishes commitments for their secret key k, that a
-client C has access to.
+call this "Single-Issuer Single-Verifier" (SISV).
 
 When S wants to issue tokens to C, they invoke the issuance protocol
-where C generates their own inputs and S uses their secret key k. In
+where C generates their own inputs, and S uses their secret key skS. In
 this setting, C can only perform token redemption with S. When a token
 redemption is required, C and S invoke the redemption phase of the
 protocol, where C uses an issued token from a previous exchange, and S
-uses k as their input again.
-
-In SISV, C proves that S has attested to the honesty of C at some point
-in the past (without revealing exactly when). S can use this information
-to inform it's own decision-making about C without having to recompute
-the trust attestation task again.
+uses skS to validate the redemption.
 
 ## Single-Issuer Forwarding-Verifier {#sifv}
```

Comment:
nit: Proxied-Verifier, maybe?

Reply:
Small conflict with abbreviation of public verifier. Would "Delegated-Verifier" work?